op-batcher/batcher/driver.go

```diff
@@ -331,6 +331,18 @@ func (l *BatchSubmitter) checkLeaderElectionBatcherIsLeaderStatus() (bool, error
 		return l.isLeader, nil
 	}
 
+	sysConf, err := bindings.NewSystemConfigCaller(l.Rollup.L1SystemConfigAddress, l.L1Client)
```

Comment:
Maybe add a comment to explain what this does or put this code inside a function?